app/models/round_assignment.rb

```diff
@@ -6,4 +7,15 @@ class RoundAssignment < ActiveRecord::Base
   validates_associated :round
   validates_associated :player
   validates_associated :target
+
+  def send_text_new_targets
+    @client = Twilio::REST::Client.new
+    if self.player.user.phone_number != nil && self.player.user.phone_number != ""
```

Comment:
Line is too long. [82/80]<br>Prefer `!expression.nil?` over `expression != nil`.<br>Redundant `self` detected.